cmd/service/utils.go

```diff
@@ -31,8 +35,32 @@ func handleError(err error) {
 	}
 }
 
+func gitClone(url string) (path string, err error) {
+	tmpFile := "/tmp/mesg-templates"
+	err = os.MkdirAll(tmpFile, os.ModePerm)
+	if err != nil {
+		return
+	}
+	path, err = ioutil.TempDir(tmpFile, string(time.Now().UnixNano()))
+	if err != nil {
+		return
+	}
+	s := cmdUtils.StartSpinner(cmdUtils.SpinnerOptions{Text: "Fetching service..."})
+	defer s.Stop()
+	_, err = git.PlainClone(path, false, &git.CloneOptions{
+		URL: url,
+	})
+	return
+}
+
 func loadService(path string) (importedService *service.Service) {
-	importedService, err := service.ImportFromPath(path)
+	var err error
+	if _, err = url.ParseRequestURI(path); err == nil {
+		path, err = gitClone(path)
```

Comment:
The new `path` is not forward to the function `buildDockerImage` thus it cannot build it..